presto-main/src/main/java/com/facebook/presto/operator/aggregation/ApproximateSetAggregation.java

```diff
@@ -30,7 +30,7 @@
 @AggregationFunction("approx_set")
 public final class ApproximateSetAggregation
 {
-    private static final int NUMBER_OF_BUCKETS = 4096;
+    public static final int NUMBER_OF_BUCKETS = 4096;
```

Comment:
nit: instead of exposing the number of buckets, let's just create a static method here called newHyperLogLog() that returns a new instance.